src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurSaeCreateRequestedCashAdvanceFileServiceImpl.java

```diff
@@ -0,0 +1,139 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import org.apache.commons.lang.StringUtils;
```

Comment:
we should use the org.apache.commons.lang3.StringUtils version of this library

Reply:
Changed.